app/components/ContextMenu.tsx

```diff
@@ -176,6 +177,37 @@ const OptionViewArtist = React.memo<{
   )
 })
 
+const OptionShuffleArtist = React.memo<{
+  artistId?: string
+}>(({ artistId }) => {
+  if (!artistId) {
```

Comment:
Another issue here is that react hooks can't be called conditionally, so this conditional above it causes an issue.

Reply:
Fixed.